### PR TITLE
WC2-363 Add django-admin command to delete entities...

### PR DESCRIPTION
**Pull request by bramj**
and form submissions for a given account.

Useful to clean up test/staging environments or prod environments right before going live.

Related JIRA tickets : https://bluesquare.atlassian.net/browse/WC2-363

### CODE REVIEW

File: iaso/management/commands/delete_all_instances_and_entities.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+            print("Deleting entities...")
+            self.delete_resources(
+                "\tEntities",
+                entities_to_delete,
+                options,
+            )
+            print("DONE.")
+
+            print("Deleting storages...")
```
Comment (bramj) [inline iaso/management/commands/delete_all_instances_and_entities.py:75]: @madewulf Should we clean the storages or not really needed?

Comment (madewulf) [reply iaso/management/commands/delete_all_instances_and_entities.py:75]: I think we should, indeed

Comment (madewulf) [reply iaso/management/commands/delete_all_instances_and_entities.py:75]: Other than that, this LGTM. Did you have the opportunity to test it on a copy of their database and check that it was not failing due to some foreign key constraints?

**Review by madewulf — APPROVED** [review]

File: iaso/management/commands/delete_all_instances_and_entities.py (hunk 1 above)
Comment (bramj) [reply iaso/management/commands/delete_all_instances_and_entities.py:75]: Just tested on a copy of their prod database, no issues. Merging!

**Pull request merged**